end-to-end-computer-vision/data/.gitignore

```diff
@@ -1,4 +0,0 @@
-# Ignore everything in this directory
```

Comment:
i think we should leave this in to make sure the directory is cloned

Reply:
Yeah, I didnt mean to remove that. Must of happened when i cleared the dir of the data images

Reply:
Added file back